guides/SPI.md

```diff
@@ -18,9 +18,22 @@ Once installed and enabled on a site, this provider will be called to fetch appl
 
 ## Setup
 In order to become a _service provider_, you have two options:
-* Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
+* Register an application in the Wix Developers Center, and add the integration component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+### Configure Integration Component in Dev Center
+In order to enable Wix applications to communicate with your app, add configurations for your component:
+1. In the side menu (under **Build your app**), click **Components** .
```

Comment:
Suggestion: You can lose the parentheses here. What you have in them is wayfinding information, which is pretty important. Enclosing them in parentheses downplays the importance.